services/conan/conan-version-helpers.js

```diff
@@ -0,0 +1,21 @@
+import yaml from 'js-yaml'
+import { NotFound, InvalidResponse } from '../index.js'
+import { latest } from '../version.js'
+
+export function parseLatestVersionFromConfig(configYaml) {
+  let versions
+  try {
+    const config = yaml.load(configYaml)
+    versions = Object.keys(config.versions)
+  } catch (err) {
+    throw new InvalidResponse({
+      prettyMessage: 'invalid config.yml',
+      underlyingError: err,
+    })
+  }
+  const version = latest(versions)
+  if (version == null) {
+    throw new NotFound({ prettyMessage: 'no versions found' })
+  }
+  return versions[versions.length - 1]
```

Comment:
I gather we can count on the versions always being ordered properly such that the last is the "latest"?

Reply:
Oops, no, this was a typo. Good catch.